newIDE/app/src/Profile/ProfileDetails.js

```diff
@@ -17,25 +19,61 @@ type Props = {
 
 export default ({ profile, onEditProfile }: Props) => {
   return profile ? (
-    <Column>
-      <Line alignItems="center">
-        <Avatar src={getGravatarUrl(profile.email || '', { size: 40 })} />
-        <Spacer />
-        <Text size="title">{profile.username}</Text>
-      </Line>
-      <Line alignItems="center">
-        <Text>
-          <Trans>You are connected as {profile.email}</Trans>
-        </Text>
-      </Line>
-      <Line justifyContent="center">
-        <RaisedButton
-          label={<Trans>Edit my profile</Trans>}
-          primary
-          onClick={onEditProfile}
-        />
-      </Line>
-    </Column>
+    <I18n>
+      {({ i18n }) => (
+        <Column>
+          <Line alignItems="center">
+            <Avatar src={getGravatarUrl(profile.email || '', { size: 40 })} />
+            <Spacer />
+            <Text
+              size="title"
+              style={{
+                color: profile.username ? 'black' : '#888',
```

Comment:
Risky as could not work well with a future theme (even if gray is fairly safe, black is not and won't show well on dark themes).
Instead, keep the default color and use something like `opacity: 0.8` to dim a bit the text.

Reply:
Oh yes, I forgot about the themes!